src/list-view.js

```diff
@@ -0,0 +1,16 @@
+(function(exports){
+  var View = function(list){
+    this.list = list;
+  }
+
+  View.prototype.displayHeadlines = function() {
+    return "<ul>" +
+
+    this.list.returnAllStories().map( story => "<li id=\"" + story.id + "\"><img src=\"" + story.thumbnail + "\"></br></br><a href=\"#\" onclick=\"readStory(" + story.id + ")\">" + story.getTitle() + "</a></li></br></br>").join("") + "</ul>"
```

Comment:
I like this approach - but be careful about putting it all on one line. It's more readable if you break it up. Also, remember you can use double and single quotes in javascript, so then you don't need so many escape characters (which also make it hard to read). e.g something like:

```
this.list.returnAllStories().map((story) => {
   "<li id='" + story.id + "'>" +
      "<img src='" + story.thumbnail + "'>" +
      "</br></br>" +
      "<a href='#' onclick='readStory(" + story.id + ")'>" + story.getTitle() + "</a>"
   "</li>").join("");
});
```
   
I can look at that and it's more obvious what the output will be as it's formatted more like I would format raw HTML.